MapboxNavigation/InstructionsBannerViewLayout.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+
+extension InstructionsBannerView {
+    
+    func commonInit() {
+        setupViews()
+        setupLayout()
+        setupAvailableBounds()
+    }
+    
+    func setupViews() {
+        backgroundColor = .clear
+        
+        let maneuverView = ManeuverView()
+        maneuverView.backgroundColor = .clear
+        maneuverView.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(maneuverView)
+        self.maneuverView = maneuverView
+        
+        let distanceLabel = DistanceLabel()
+        distanceLabel.translatesAutoresizingMaskIntoConstraints = false
+        distanceLabel.adjustsFontSizeToFitWidth = true
+        distanceLabel.minimumScaleFactor = 0.2
+        addSubview(distanceLabel)
+        self.distanceLabel = distanceLabel
+        
+        let primaryLabel = PrimaryLabel()
+        primaryLabel.translatesAutoresizingMaskIntoConstraints = false
+        primaryLabel.adjustsFontSizeToFitWidth = true
+        primaryLabel.numberOfLines = 3
+        primaryLabel.minimumScaleFactor = 0.5
+        primaryLabel.lineBreakMode = .byTruncatingTail
+        addSubview(primaryLabel)
+        self.primaryLabel = primaryLabel
+        
+        let secondaryLabel = SecondaryLabel()
+        secondaryLabel.translatesAutoresizingMaskIntoConstraints = false
+        secondaryLabel.adjustsFontSizeToFitWidth = true
+        secondaryLabel.numberOfLines = 3
```

Comment:
This should be `2`

Reply:
Using minimum font scale 20/26 would result in only 18 pts remaining so it would never render `2` lines of text in the secondary instruction label.